src/jit/codegenlinear.cpp

```diff
@@ -1075,6 +1022,8 @@ void CodeGen::genCheckConsumeNode(GenTree* treeNode)
 //    of the multi-reg return.
 regNumber CodeGen::genConsumeReg(GenTree* tree)
 {
+    assert(!tree->isContained());
```

Comment:
No, it would not: the caller goes through `genConsumeRegs(GenTree* tree)`, which is a no-op if `tree` is a contained spill temp.